app/utils/paginate.js

```diff
@@ -0,0 +1,43 @@
+async function paginate(model, query, options) {
+  const opts = { limit: 25, page: 1, url: process.env.SITE_URL, ...options };
```

Comment:
Is the page always 1?

Reply:
Ok, I see it will be overwritten by `options`. This `options` param cannot be literal. It should be sanitized in some code layer, maybe in the validations, maybe here in the service, but it cannot be literal from the request because of SQL injection vulnerabilities.
I suggest adding schema validation for pagination query params.